internal/bminventory/inventory_v2_handlers.go

```diff
@@ -130,6 +130,10 @@ func (b *bareMetalInventory) V2GetClusterInstallConfig(ctx context.Context, para
 		return common.GenerateErrorResponder(fmt.Errorf("Failed to get cluster %s: %w", params.ClusterID, err))
 	}
 
+	if common.IsDay2Cluster(cluster) {
+		return common.GenerateErrorResponderWithDefault(fmt.Errorf("Cannot get install config for Day2 cluster %s", params.ClusterID), http.StatusBadRequest)
```

Comment:
@avishayt I'm not sure this is a good message to return to the user, WDYT?

Reply:
I personally don't like exposing "day2 cluster" to users, as it has no meaning.  I would say:
"the install config is not available because this cluster resource is used only for adding additional hosts to an existing cluster"